server/utils/placementRequests/checkYourAnswersUtils.test.ts

```diff
@@ -20,47 +21,78 @@ describe('checkYourAnswersUtils', () => {
         },
-        rows: pageResponsesAsSummaryListItems(placementApplication, 'same-ap'),
+        rows: pageResponsesAsSummaryListItems(placementApplication, 'same-ap', application),
       })
     })
-    describe('pagesForReview', () => {
-      it('should return all of the pages except the check-your-answers page', () => {
-        placementApplication = addResponseToFormArtifact(placementApplication, {
-          section: 'request-a-placement',
-          page: 'check-your-answers',
-          key: 'reviwed',
-          value: '1',
-        }) as PlacementApplication
-        expect(pagesForReview(placementApplication)).toEqual(['same-ap'])
-      })
+  })
+  describe('pagesForReview', () => {
+    it('should return all of the pages except the check-your-answers page', () => {
+      placementApplication = addResponseToFormArtifact(placementApplication, {
+        section: 'request-a-placement',
+        page: 'check-your-answers',
+        key: 'reviwed',
+        value: '1',
+      }) as PlacementApplication
+      expect(pagesForReview(placementApplication)).toEqual(['same-ap'])
     })
+  })
 
-    describe('pageResponsesAsSummaryListItems', () => {
-      it('should return the correct object for the summary card', () => {
-        expect(pageResponsesAsSummaryListItems(placementApplication, 'same-ap')).toEqual([
-          {
-            key: { text: 'Do you want this person to stay in the same Approved Premises (AP)?' },
-            value: { text: 'Yes' },
-            actions: {
-              items: [
-                {
-                  href: paths.placementApplications.pages.show({
-                    task: 'request-a-placement',
-                    page: 'same-ap',
-                    id: placementApplication.id,
-                  }),
-                  text: 'Change',
-                  visuallyHiddenText: 'Do you want this person to stay in the same Approved Premises (AP)?',
-                },
-              ],
-            },
+  describe('pageResponsesAsSummaryListItems', () => {
+    it('should return the correct object for the summary card', () => {
+      expect(pageResponsesAsSummaryListItems(placementApplication, 'same-ap', application)).toEqual([
+        {
+          key: { text: 'Do you want this person to stay in the same Approved Premises (AP)?' },
+          value: { text: 'Yes' },
+          actions: {
+            items: [
+              {
+                href: paths.placementApplications.pages.show({
+                  task: 'request-a-placement',
+                  page: 'same-ap',
+                  id: placementApplication.id,
+                }),
+                text: 'Change',
+                visuallyHiddenText: 'Do you want this person to stay in the same Approved Premises (AP)?',
+              },
+            ],
           },
-        ])
-      })
+        },
+      ])
     })
   })
+
+  it('if the page name is "additional-documents"', () => {
```

Comment:
I feel like this stuff should be done after the additional documents page is added, as it doesn't make sense linerarly otherwise